microsoft/testsuites/performance/storageperf.py

```diff
@@ -322,7 +322,7 @@ def _perf_premium_datadisks(
             disk_type=DiskType.premiumssd,
             numjob=core_count,
             block_size=block_size,
-            size_gb=8,
+            size_mb=8192,
```

Comment:
will it affect premium storage perf testing？

Reply:
It should not affect it. It is the same size just specified in MB now rather than GB. Btw is this value passed to database?

Reply:
no, db doesn't store this value